### DISCUSSION TITLE
Server redirect

### DISCUSSION THREAD

**Discussion by DiogoMurano** [jeremylvln/Shulker#613] [Ideas]
Why not release the shulker redirect feature to move the player between servers?

The redis `shulker:teleport` event already exists and is implemented, it would just be necessary to expose it in the ShulkerProxyAPI

**Comment by jeremylvln** [comment]
Hi @DiogoMurano,

That's a fair point, and can be easily added too, will do shortly!

**Reply by jeremylvln** [reply]
That's now part of the 0.10.0 release!

Do not hesitate to give me feedbacks :)
